### PR TITLE
Optional tracy/tracy dependency

### PR DESCRIPTION
**Pull request by flokixdev**

### CODE REVIEW

File: src/Search.php [hunk 1]
```diff
@@ -62,7 +62,9 @@ public function __construct(Engine $engine, Router $router)
 	 */
 	public function search(string $query)
 	{
-		Debugger::timer('search_request');
+		if(class_exists(Debugger::class)) {
```
Comment (janbarasek) [inline src/Search.php]: Codestyle should be:

```suggestion
		if (class_exists('\Tracy\Debugger') === true) {
```

Comment (flokixdev) [reply src/Search.php]: @janbarasek Why `=== true`? the return type is non-nullable bool

**Comment by janbarasek** [comment]
I think better codestyle is `class_exists('\Tracy\Dumper')` instead of `class_exists('Tracy\Dumper')`. I will fix it in future.

Thanks!

File: src/Search.php (hunk 1 above)
Comment (janbarasek) [reply src/Search.php]: It's only my codestyle. I think you can keep your version, it does not matter.

Thanks.

**Pull request merged**